src/cljs/athens/views/blocks/textarea_keydown.cljs

```diff
@@ -375,7 +375,9 @@
     (when (empty? selected-items)
       (if shift
         (dispatch [:unindent editing-uid d-key-down])
```

Comment:
@neotyk We both missed this `:unindent`

Reply:
I've already ported it :-)

Reply:
Yeah, you're right, I didn't check port this usage 🙇‍♂️